### PR TITLE
clarify dependency installation for webpack

### PR DESCRIPTION
**Pull request by applegrain**
Clarifies what commands to run for developers that aren't familiar with browserify/webpack.

### CODE REVIEW

File: docs/docs/getting-started.md [hunk 1]
```diff
@@ -27,6 +27,12 @@ ReactDOM.render(
 );
 ```
 
+If you are using webpack to bundle your dependencies: 
+
+```sh
+$ npm install --save react react-dom babelify babel-preset-react
```
Comment (zpao) [inline docs/docs/getting-started.md]: We have this exact line just below here (though it's in the browserify block). Is that what you intended?

As a side note, we could definitely make this section a little clearer, adding a shared dependency section (eg installing react-dom) and then a section with specific instructions for browserify and another for browserify.

**Comment by applegrain** [comment]
@zpao I must have missed that! My intention was to separate webpack and browserify instructions because a few students (using webpack) I have pointed to this page have been confused over what commands to run. 

>As a side note, we could definitely make this section a little clearer, adding a shared dependency section (eg installing react-dom) and then a section with specific instructions for browserify and another for browserify.

I'll rework this PR and see if I can make this section a bit clearer.

**Comment by zpao** [comment]
The fact that you missed it goes to show how much room for improvement there is :) We appreciate the help!

**Comment by applegrain** [comment]
PR to replace this one: https://github.com/facebook/react/pull/5658

**Pull request closed without merging**